CHANGELOG.md

```diff
@@ -4,19 +4,23 @@
 * None
 
 ### Enhancements
-* None
+* Allow collections of non-embedded links in asymmetric objects. ([realm/realm-core#7003](https://github.com/realm/realm-core/pull/7003))
```

Comment:
```suggestion
* Allow asymmetric objects to contain fields with non-embedded (top-level) links (rather than only embedded links). ([realm/realm-core#7003](https://github.com/realm/realm-core/pull/7003))
```